secboot/secboot_sb_test.go

```diff
@@ -32,9 +32,14 @@ import (
 	"os"
 	"path/filepath"
 
+	. "gopkg.in/check.v1"
+
 	"github.com/canonical/go-tpm2"
+	"github.com/canonical/go-tpm2/linux"
```

Comment:
Does this build ok? github.com/canonical/go-tpm2/linux only provides a single API - a transmission interface for the Linux TPM character device, which is used internally by secboot. I'd be surprised if this were required in snapd for testing.

Reply:
It is used inside a mock https://github.com/snapcore/snapd/pull/10715/files/81ee7783f70761a6819fb337778997710f188814#r703699848